x-pack/plugins/apm/dev_docs/telemetry.md

```diff
@@ -19,31 +19,27 @@ to the telemetry cluster using the
 During the APM server-side plugin's setup phase a
 [Saved Object](https://www.elastic.co/guide/en/kibana/master/managing-saved-objects.html)
 for APM telemetry is registered and a
-[task manager](../../task_manager/server/README.md) task is registered and started.
+[task manager](../../task_manager/README.md) task is registered and started.
 The task periodically queries the APM indices and saves the results in the Saved
 Object, and the usage collector periodically gets the data from the saved object
 and uploads it to the telemetry cluster.
 
 Once uploaded to the telemetry cluster, the data telemetry is stored in
 `stack_stats.kibana.plugins.apm` in the xpack-phone-home index.
 
-### Generating sample data
+### Collect a new telemetry field
 
-The script in `scripts/upload_telemetry_data` can generate sample telemetry data and upload it to a cluster of your choosing.
+In order to collect a new telemetry field you need to add a task which performs the query that collects the data from the cluster.
 
-You'll need to set the `GITHUB_TOKEN` environment variable to a token that has `repo` scope so it can read from the
-[elastic/telemetry](https://github.com/elastic/telemetry) repository. (You probably have a token that works for this in
-~/.backport/config.json.)
+All the available tasks are [here](../server/lib/apm_telemetry/collect_data_telemetry/tasks.ts)
```

Comment:
Might be better to use a versioned link in case that file is moved and we forget to update the docs
```suggestion
All the available tasks are [here](https://github.com/elastic/kibana/blob/ba84602455671f0f6175bbc0fd2e8f302c60bbe6/x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts)
```